models/shipping-method/src/shipping-rate/shipping-rate-draft/generator.ts

```diff
@@ -0,0 +1,15 @@
+import { CentPrecisionMoney } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TShippingRateDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/shippingMethods#shippingratedraft
+
+const generator = Generator<TShippingRateDraft>({
+  fields: {
+    price: fake(() => CentPrecisionMoney.random()),
+    freeAbove: fake(() => CentPrecisionMoney.random()),
```

Comment:
The documentation says this one should be `Money`
![image](https://github.com/commercetools/test-data/assets/8136478/08fee78c-124a-4c58-ab61-a4fd2260872f)